script/testdata/spawn_callee_current_cycles.c

```diff
@@ -0,0 +1,15 @@
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "ckb_syscalls.h"
+
+int main(int argc, char *argv[]) {
+  int caller_cycles = atoi(argv[0]);
+  // Callee's current cycles must >= caller's current cycles.
+  int callee_cycles = ckb_current_cycles();
+  if (callee_cycles < caller_cycles) {
+    return 1;
```

Comment:
The "gap" between "callee_cycles" and "callee_cycles" can be estimated. More strict checking can be used.